### DISCUSSION TITLE
Add Graphiql

### DISCUSSION THREAD

**Discussion by MuhammadRafeh** [graphql/graphql-http#95] [Q&A] [answered]
express-graphql had a Graphiql. And that module is now deprecated. I want that feature in this module.

**Comment by MuhammadRafeh** [comment]
do any one one know alternative module ??? which contain graphiql?

**Comment by benjie** [comment]
You can install GraphiQL alongside graphql-http using a middleware in your server. As an example, here’s a GraphiQL distribution I wrote that’s easy to use like this: https://www.npmjs.com/package/ruru/v/2.0.0-alpha.6

**Comment by enisdenjo** [comment, marked as answer]
This library implements only the GraphQL over HTTP spec, please see the [disclaimer in the readme](https://github.com/graphql/graphql-http#only-graphql-over-http).

For a fully featured GraphQL server - I recommend looking into [GraphQL Yoga](https://the-guild.dev/graphql/yoga-server).

Also see #38 and #87.
